app/realm/actions/client_upgrade.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ClientUpgradeTypes} from 'app/realm/action_types';
+
+export function setLastUpgradeCheck() {
```

Comment:
Following the code changes I’m not sure what is the role of this. Also this does not need to be a thunk

Reply:
The common practice seems to be to write the actions as functions in R3. Since I'm not returning anything though, I can just write this as an arrow function to dispatch the update?

Reply:
yes you are right, that is the common practice BUT cause normally you'll be performing some sort of async executing which I don't think is needed at all here.